internal/controller/datadogagent/feature/enabledefault/const.go

```diff
@@ -0,0 +1,12 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package enabledefault
+
+const (
+	DDAgentDaemonSet             = "AGENT_DAEMONSET"
+	DDClusterAgentDeployment     = "CLUSTER_AGENT_DEPLOYMENT"
+	DDDatadogAgentCustomResource = "DATADOG_AGENT_CR_NAME"
```

Comment:
```suggestion
	DDAgentDaemonSet             = "NODE_AGENT_DAEMONSET"
	DDClusterAgentDeployment     = "CLUSTER_AGENT_DEPLOYMENT"
	DDDatadogAgentCustomResource = "DATADOGAGENT_CR_NAME"
```

Reply:
Discussed offline, will keep `AGENT_DAEMONSET` as it's already referenced by Helm and within the Agent codebase.
Modified `DATADOGAGENT_CR_NAME`